go/arrow/tensor/tensor.go

```diff
@@ -104,6 +104,7 @@ func (tb *tensorBase) NumDims() int             { return len(tb.shape) }
 func (tb *tensorBase) DimName(i int) string     { return tb.names[i] }
 func (tb *tensorBase) DataType() arrow.DataType { return tb.dtype }
 func (tb *tensorBase) Data() *array.Data        { return tb.data }
+func (tb *tensorBase) DimNames() []string     { return tb.names }
```

Comment:
please `gofmt` your code.
it makes for nice, regular looking, code. (for some definition of "nice". thankfully, all gophers have _now_ the same definition. :P)

a simple:
```
$> cd github.com/apache/arrow/go
$> gofmt -w .
```
should do.

it's a good idea to wire `gofmt` inside your favorite editor (or, even better, use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) that does pretty much the same thing than `gofmt` but also adds/removes automatically package to/from the `import` block when used/unused.)
see:
- https://github.com/golang/go/wiki/IDEsAndTextEditorPlugins

Reply:
> please `gofmt` your code.
> it makes for nice, regular looking, code. (for some definition of "nice". thankfully, all gophers have _now_ the same definition. :P)
> 
> a simple:
> 
> ```
> $> cd github.com/apache/arrow/go
> $> gofmt -w .
> ```
> should do.
> 
> it's a good idea to wire `gofmt` inside your favorite editor (or, even better, use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) that does pretty much the same thing than `gofmt` but also adds/removes automatically package to/from the `import` block when used/unused.)
> see:
> 
> * https://github.com/golang/go/wiki/IDEsAndTextEditorPlugins

Thanks for the suggestion! I've configured my GoLand File Watcher to do "go fmt".